apps/system/test/marionette/task_manager_test.js

```diff
@@ -179,6 +179,23 @@ marionette('Task Manager', function() {
     });
   });
 
+  suite('new sheet buttons', function() {
+    setup(function() {
+      taskManager.show();
+    });
+
+    test('should open a new sheet', function() {
+      taskManager.newSheetButton.tap();
+      system.waitForBrowser('app://search.gaiamobile.org/newtab.html');
+    });
+
+    test('should open a new private sheet', function() {
+      taskManager.newPrivateSheetButton.tap();
+      system.waitForBrowser(
+        'app://search.gaiamobile.org/newtab.html?private=1');
+    });
```

Comment:
we should assert that the appWindow has the `.private` class here.
It's not perfect but it'll be inline with the other private browsing tests.